src/Install/Steps/StoreConfig.php

```diff
@@ -55,10 +55,13 @@ private function buildConfig()
     {
         return [
             'debug'    => $this->debugMode,
-            'poweredByHeader'  => true,
             'database' => $this->dbConfig->toArray(),
             'url'      => (string) $this->baseUrl,
             'paths'    => $this->getPathsConfig(),
+            'headers'  => [
+                'poweredByHeader'  => true,
+                'referrerPolicy'
```

Comment:
Not sure if I misunderstand what this is, but should we maybe add the default value of this from `ReferrerPolicyHeader.php`?